.babelrc

```diff
@@ -16,5 +16,24 @@
         "loose": true
       }
     ]
-  ]
+  ],
+
+  "env": {
+    "development": {
+      "plugins": ["external-helpers"]
+    },
+
+    "production": {
+      "plugins": [
+        "external-helpers",
+        "transform-react-inline-elements",
+        "transform-react-constant-elements",
```

Comment:
I'm still very confuse about this inlining thing. For me, both examples are the same, they're invariants so it's just a matter of different syntax. Modern JIT inline invariant anyway so I don't really see the point.
Let's leave it as is for now if it's a good practice. Whatever it does or doesn't won't impact much the performance of the app.